### PR TITLE
Inter is dead long live inter

### PR DESCRIPTION
**Pull request by superdump**

### CODE REVIEW

File: owr/owr_inter_sink.c [hunk 1]
```diff
@@ -0,0 +1,237 @@
+/*
+ * GStreamer
```
Comment (superdump) [inline owr/owr_inter_sink.c]: OwrInterSink or something else that is consistent with other files?

File: owr/owr_inter_sink.c [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+static GstStaticPadTemplate sink_template = GST_STATIC_PAD_TEMPLATE ("sink",
+    GST_PAD_SINK,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS_ANY
+    );
+
+#define parent_class owr_inter_sink_parent_class
+G_DEFINE_TYPE (OwrInterSink, owr_inter_sink, GST_TYPE_ELEMENT);
+
+static gboolean owr_inter_sink_sink_query (GstPad * pad, GstObject * parent,
```
Comment (superdump) [inline owr/owr_inter_sink.c]: I don't like this _sink_sink_ naming. Just one sink is fine I think, given that the sink doesn't have any src pads. :smile:

File: owr/owr_inter_sink.c [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "owr_inter_sink.h"
+
+#define GST_CAT_DEFAULT owr_inter_sink_debug
+GST_DEBUG_CATEGORY_STATIC (GST_CAT_DEFAULT);
```
Comment (superdump) [inline owr/owr_inter_sink.c]: This looks like gst-indent style. We use WebKit style throughout the project. This script installs it: https://github.com/EricssonResearch/openwebrtc-build-scripts/blob/master/bootstrap/build-webkit-style-check.sh . Note that it copies some files from that repo into the installation destination too. It should end up somewhere in ~/.openwebrtc.

We really need to replicate that style using some easier to install tool...

File: owr/owr_inter_sink.c (hunk 2 above)
Comment (superdump) [inline owr/owr_inter_sink.c]: Ah, I see that there are references to the 'otherpad' src pad.

File: owr/owr_inter_src.c [hunk 4]
```diff
@@ -0,0 +1,192 @@
+/*
+ * GStreamer
```
Comment (superdump) [inline owr/owr_inter_src.c]: Same here.

File: owr/owr_inter_src.c [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+ */
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "owr_inter_src.h"
+
+#define GST_CAT_DEFAULT owr_inter_src_debug
+GST_DEBUG_CATEGORY_STATIC (GST_CAT_DEFAULT);
```
Comment (superdump) [inline owr/owr_inter_src.c]: This is GStreamer style.

**Comment by superdump** [comment]
Apart from style and having file headers consistent with the rest of OpenWebRTC, I think this looks good.

**Comment by superdump** [comment]
I have fixes for the style issues locally.

**Comment by Rugvip** [comment]
:+1:
But we should write some tests as well, with owr elements and some different gst pipelines.

**Comment by sdroege** [comment]
Any existing tests are covering this already

**Comment by sdroege** [comment]
@superdump please merge with your style fixes then :) Thanks for cleaning up the style

**Pull request closed without merging**